lstchain/mc/sensitivity.py

```diff
@@ -71,40 +72,45 @@ def process_mc(dl1_file, dl2_file, mc_type):
     sim_par = read_sim_par(dl1_file)
     events = pd.read_hdf(dl2_file)
```

Comment:
It needs "key=dl2_params_lstcam_key"?

Reply:
Ah yes, for reading the params is not necessary, but for events it needs the key. I have it like this:
sim_par = read_sim_par(dl1_file)                              
events = pd.read_hdf(dl2_file, key='dl2/event/telescope/parameters/LST_LSTCam')

Reply:
yes, thanks!